src/main/kotlin/racing/Game.kt

```diff
@@ -0,0 +1,23 @@
+package racing
+
+class Game(
+    numberOfCar: Int,
+    private val tryCount: Int
+) {
+    private val carList = List(numberOfCar) { Car() }
```

Comment:
```suggestion
    private val cars = List(numberOfCar) { Car() }
```
필드에 대한 네이밍은 컬렉션에 의존하기보다는 "복수의 값"라는 것을 표현하는 것이 좋습니다.